packages/progress-indicator/src/ProgressIndicatorCollapseButton.tsx

```diff
@@ -37,7 +37,7 @@ export const ProgressIndicatorCollapseButton = ({
 	const title = useMemo(() => {
 		const activeItemIndex = items.findIndex(({ status }) => status === 'doing');
 		if (activeItemIndex === -1) return 'Progress';
-		return `Doing step ${activeItemIndex + 1} of ${items.length}`;
+		return `${activeItemIndex + 1} of ${items.length} steps completed`;
```

Comment:
This calculation is incorrect. What you have in `packages/progress-indicator/src/ProgressIndicator.tsx` is correct.